src/test/scala/io/citrine/lolo/util/LoloPyDataLoaderTest.scala

```diff
@@ -0,0 +1,111 @@
+package io.citrine.lolo.util
+import io.citrine.lolo.TestUtils
+import io.citrine.lolo.learners.RandomForest
+import io.citrine.lolo.stats.functions.Friedman
+import org.junit.Test
+
+class LoloPyDataLoaderTest {
+
+  private def getData: Array[Byte] = {
+    val stream = getClass.getClassLoader.getResourceAsStream("4by4double.bytes")
+    val byteArray = new Array[Byte](128)
+    stream.read(byteArray)
+    byteArray
+  }
+
+  @Test
+  def testFeatureArray(): Unit = {
+    val data = LoloPyDataLoader.getFeatureArray(getData, 4, bigEndian = false)
+    assert(data.length == 4)
+    assert(data.head.length == 4)
+    assert(Math.abs(data.head(1) - 1) < 1e-6, "Array read in wrong order")
```

Comment:
Could you test the other values?  This test is insensitive to transposes.

Reply:
This test should be sensitives to transposes because the array is [[0, 1, 2, 3], [4, ...]]. But, for my own paranoia's sake, I'll add in another check